WorkloadManagementSystem/Agent/test/Test_Agent_WorkloadManagementSystem.py

```diff
@@ -1,6 +1,8 @@
 """ Test class for WMS agents
 """
 
+# pylint: disable=protected-access,missing-docstring
```

Comment:
is missing-docstring an error we should allow ourselves to ever disable ? IMHO, no :-)

Reply:
In test files, yes!

Reply:
the first line of the docstring is used by pytest as an explanation of the test. This makes the test output very readable.